WooCommerce/Classes/ViewRelated/Dashboard/Onboarding/StoreOnboardingViewModel.swift

```diff
@@ -50,16 +58,15 @@ class StoreOnboardingViewModel: ObservableObject {
 
     private var state: State
 
-    private let placeholderTasks: [StoreOnboardingTaskViewModel] = Array(repeating: StoreOnboardingTaskViewModel.placeHolder(),
-                                                                         count: 3)
+    private let placeholderTasks: [StoreOnboardingTaskViewModel] = [.placeHolder(), .placeHolder(), .placeHolder()]
```

Comment:
just curious, are these diffs to fix any issue or just shortening the number of lines?

Reply:
Oh, I changed this to resolve a duplicate UUID SwiftUI error in af670644b6a494086cc6914864ab303490bfff8c . I explained it in the commit message.

Reply:
interesting, what kind of error did it show?

Reply:
The following error is logged when the view is in redacted mode.

```
ForEach<Array<StoreOnboardingTaskViewModel>, UUID, _ConditionalContent<ModifiedContent<StoreOnboardingTaskView, Shimmer>, StoreOnboardingTaskView>>: the ID 106AB293-710F-4174-B938-FC1E90E5BC9A occurs multiple times within the collection, this will give undefined results!
```

Reply:
ooh nice catch!